src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralConcluder.kt

```diff
@@ -35,13 +35,15 @@ class ReferralConcluder(
     }
   }
 
+  fun requiresEosr(referral: Referral): Boolean {
+    return getNumberOfAttendedAppointments(referral) > 0
```

Comment:
And it's concluded, right?

Reply:
yes

Reply:
Can we add those conditions?

Reply:
Added a checked that the referral has infact ended for some reason

Reply:
Agreed logic ... true if:

endOfServiceReport does not exist 
and
(allSessionsDelivered) || (endRequested && atLeastOneSessionDelivered)